events.html

```diff
@@ -0,0 +1,212 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>About Us | BAMP</title>
+        <link rel="stylesheet" href="style/index.css">
+        <script src="scripts/index.js" type="module" defer></script>
+    </head>
+
+    <body>
+        <div class="header">
+            <a class="header--linkLogo" href="/index.html">
+                <img class="header--logo" src="images/logo-mini-color.svg">
+            </a>
+            <a class="header--link" href="/artwork.html">Artwork</a>
+            <a class="header--link" href="/about_us.html">About Us</a>
+            <a class="header--link" href="/programs.html">Programs</a>
+            <a class="header--link" href="/events.html">Events</a>
+            <a class="header--link" href="/services.html">Services</a>
+            <a class="header--link" href="#">Shop</a>
+            <a class="header--link" href="/get_involved.html">Get Involved</a>
+            <a class="header--link header--link-tight" href="#"><span class="btnLink btnLink-small">Get Updates</span></a>
+            <a class="header--link header--link-tight" href="#"><span class="btnLink btnLink-red btnLink-small">Donate</span></a>
+            <button id="hamburger" class="header--hamburger">☰</button>
+        </div>
+    </body>
```

Comment:
I think the `</body>` tag should be the second to last (right above `</html>`) of the whole page